package.json

```diff
@@ -22,6 +22,7 @@
 		"IS_GUTENBERG_PLUGIN": true
 	},
 	"dependencies": {
+		"@ariakit/react": "0.4.5",
```

Comment:
Also we might want to update `@ariakit/test` together with `@ariakit/react`.

Reply:
Makes sense, done 👍